payments/sofort/__init__.py

```diff
@@ -56,7 +56,9 @@ def get_form(self, payment, data=None):
                 "interface_version": "django-payments",
                 "amount": payment.total,
                 "currency": payment.currency,
-                "description": payment.description,
+                "description": payment.description
+                if len(payment.description) <= 40
+                else (payment.description[:37] + "..."),
```

Comment:
**minor nitpick**: Is the `…` character accepted by Sofort? Note that it's only one character, so you can leave two more digits.